agent/src/softwarecontaineragentadaptor.cpp

```diff
@@ -73,6 +73,13 @@ bool SoftwareContainerAgentAdaptor::SetGatewayConfigs(
     return m_agent.setGatewayConfigs(containerID, configs);
 }
 
+void SoftwareContainerAgentAdaptor::ListCapabilities(std::vector<std::string> &capabilities,
+                                                     bool &success)
+{
+    capabilities = m_agent.listCapabilities();
+    success = true; // This operation can not fail
```

Comment:
That does not seem like a good reason to me.
I think there should always be a purpose to a return value otherwise users might end up getting false positives.